ecllibrary/teststd/uni/TestCountWords.ecl

```diff
@@ -9,7 +9,7 @@ EXPORT TestCountWords := MODULE
   EXPORT TestConst := MODULE
     //Check action on strings with no entries: empty source string, search string, or return string.
     EXPORT Test01 := ASSERT(Uni.CountWords(U'', U'') = 0);
-    EXPORT Test01b := ASSERT(Uni.CountWords(U'abcde', U'') = 0);
+    EXPORT Test01b := ASSERT(Uni.CountWords(U'abcde', U'') = 1);
```

Comment:
Note: This test was previously wrong - corrected test (and code) when implementing the new function